recipe/add_deps.py

```diff
@@ -0,0 +1,18 @@
+import os
+from os.path import join
+
+from bt.source import SRC_URL
+from ll.utils import download
```

Comment:
It seems like the Anaconda recipe uses `bt.source` and `ll.utils` from `root` or some other included system Python where this recipe is being built. Have no clue what these are or why they are in use. May need to replace them with something, but am not entirely sure what.

Reply:
Any ideas on this problem, @mbargull?